README.md

```diff
@@ -128,84 +105,36 @@ Failures: 1 (HIGH: 1, CRITICAL: 0)
-+---------------------------+------------+----------------------+----------+------------------------------------------+
-```
-
-</details>
-
-
-# Features
+# Highlights
 
 - Comprehensive vulnerability detection
   - OS packages (Alpine Linux, Red Hat Universal Base Image, Red Hat Enterprise Linux, CentOS, AlmaLinux, Rocky Linux, CBL-Mariner, Oracle Linux, Debian, Ubuntu, Amazon Linux, openSUSE Leap, SUSE Enterprise Linux, Photon OS and Distroless)
   - **Language-specific packages** (Bundler, Composer, Pipenv, Poetry, npm, yarn, Cargo, NuGet, Maven, and Go)
+  - High accuracy, especially [Alpine Linux][alpine] and RHEL/CentOS
+- Supply chain security (SBOM support)
+  - Support CycloneDX
+  - Support SPDX
 - Misconfiguration detection (IaC scanning) 
-  - A wide variety of built-in policies are provided **out of the box**
-    - Kubernetes, Docker, Terraform, and more coming soon
-  - Support custom policies
+  - Wide variety of security checks are provided **out of the box**
+  - Kubernetes, Docker, Terraform, and more
+  - User-defined policies using [OPA Rego][rego]
 - Secret detection
   - A wide variety of built-in rules are provided **out of the box**
-  - Support custom rules
-  - Scan container images at high speed
+  - User-defined patterns
+  - Efficient scanning of container images
 - Simple
-  - Specify only an image name, a path to config files, or an artifact name
-- Fast
-  - The first scan will finish within 10 seconds (depending on your network). Consequent scans will finish in single seconds.
-- Easy installation
-  - `apt-get install`, `yum install` and `brew install` are possible.
-  - **No pre-requisites** such as installation of DB, libraries, etc.
-- High accuracy
-  - **Especially [Alpine Linux][alpine] and RHEL/CentOS**
-  - Other OSes are also high
-- DevSecOps
-  - **Suitable for CI** such as GitHub Actions, Jenkins, GitLab CI, etc.
-- Support multiple targets
-  - container image, local filesystem and remote git repository
-- Supply chain security (SBOM support)
-  - Support CycloneDX
-  - Support SPDX
-
-# Integrations
-- [GitHub Actions][action]
-- [Visual Studio Code][vscode]
-
-# Documentation
-The official documentation, which provides detailed installation, configuration, and quick start guides, is available at https://aquasecurity.github.io/trivy/.
+  - Available in apt, yum, brew, dockerhub
+  - **No pre-requisites** such as a database, system libraries, or eny environmental requirements. The binary runs anywhere.
+  - The first scan will finish within 10 seconds (depending on your network). Consequent scans will finish instantaneously.
+- Fits your workflow
+  - **Great for CI** such as GitHub Actions, Jenkins, GitLab CI, etc.
+  - Available as extension for IDEs such as vscode, jetbrains, vim
+  - Available as extension for Docker Desktop, Rancher Desktop
+  - See [integrations] section in the documentation.
 
```

Comment:
Could we add an installation section, even if it is to link to that part of the docs since many people will simply look for "installation" in the README 
https://aquasecurity.github.io/trivy/v0.27.1/getting-started/installation/